core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/internal/signer/AwsChunkedV4PayloadSigner.java

```diff
@@ -252,4 +252,49 @@ private void setupPreExistingTrailers(ChunkedEncodedInputStream.Builder builder,
             request.removeHeader(header);
         }
     }
+
+    public static Builder builder() {
+        return new BuilderImpl();
+    }
+
+    public interface Builder {
+        Builder credentialScope(CredentialScope credentialScope);
+
+        Builder chunkSize(int chunkSize);
```

Comment:
Should probably be boxed type so that we can throw an exception or set a default if the caller doesn't set it.